core/vm/exit_code/exit_code.hpp

```diff
@@ -22,6 +22,8 @@ namespace fc::vm {
   enum class VMExitCode : int64_t {
     kFatal = -1,
 
+    kErrSerializationPre7 = 1,
```

Comment:
Duplicate value with kSysErrSenderInvalid

Reply:
This is compatible with lotus

Reply:
Ok